### PR TITLE
Improve consistency of command line names & more

### PR DESCRIPTION
**Pull request by mantognini**
Linked to epfl-lara/inox#41. Require #67 (easier than to deal with merge conflict at this point).

Removed `--inlineposts`, which wasn't used in the code at all.

Removed `--parallelvcs`, as it is now always parallel.

Fix internal links in the documentation. The doc is also updated to reflect the new option names and includes some of the missing ones (not the `--print*` options).

### CODE REVIEW

File: core/src/sphinx/gettingstarted.rst [hunk 1]
```diff
@@ -5,7 +5,7 @@ Getting Started
 
 Stainless is currently only available as a command line tool,
 which exposes most of the functionality. See the
-`installation documentation <installation.rst>`_
+:doc:`installation documentation <installation>`
```
Comment (samarion) [inline core/src/sphinx/gettingstarted.rst]: Did the previous syntax not work with sphinx? Its main advantage was that it was compatible with Github markdown, so users could navigate through some basic doc without having to go to the website.

File: frontends/scalac/src/it/scala/stainless/verification/VerificationSuite.scala [hunk 2]
```diff
@@ -62,8 +62,7 @@ class InlinedPostsVerificationSuite extends VerificationSuite {
   override def configurations = super.configurations.map {
     seq => Seq(
       inox.optSelectedSolvers(Set("smt-z3")),
-      inox.solvers.optCheckModels(true),
-      extraction.inlining.optInlinePosts(true)
```
Comment (samarion) [inline frontends/scalac/src/it/scala/stainless/verification/VerificationSuite.scala]: We ended up running into some conceptual issues when discussing this option so it was never fully implemented. Ok with removing it for now, you can also remove the `InlinedPostsVerificationSuite` while we're at it.

File: core/src/sphinx/gettingstarted.rst (hunk 1 above)
Comment (mantognini) [reply core/src/sphinx/gettingstarted.rst]: Unfortunately, they resulted in 404 error: the HTML link was against the rst file, not the generated html one. To mitigate the issue we could host a proper HTML version of the documentation somewhere.

File: frontends/scalac/src/it/scala/stainless/verification/VerificationSuite.scala (hunk 2 above)
Comment (mantognini) [reply frontends/scalac/src/it/scala/stainless/verification/VerificationSuite.scala]: Done.

File: core/src/sphinx/gettingstarted.rst (hunk 1 above)
Comment (samarion) [reply core/src/sphinx/gettingstarted.rst]: This is very unfortunate but shouldn't be a blocker for this PR. There seems to be [some work](https://gist.github.com/shimizukawa/d62d947509a24f816168) on getting sphinx and markdown to play nice, we should open an issue and investigate.

**Comment by mantognini** [comment]
Just double checking, is everything okay here?

**Comment by samarion** [comment]
It looks fine to me.

**Comment by mantognini** [comment]
Merged in 3e8aa4485761908531f49cab606806bc21f4a777.

**Pull request closed without merging**